labshare-client/src/views/Admin.vue

```diff
@@ -0,0 +1,241 @@
+<i18n>
+{
+  "en": {
+  },
+  "de": {
+    "title": "User Management"
+  }
+}
+</i18n>
+<template>
+  <div class="list-view">
+    <h1 class="mt-4">{{$t("title")}}</h1>
+    <div v-if="error" class="alert alert-danger" role="alert">{{ error }}</div>
+
+    <div style="margin-top: 40px">
+      <b-table
+        v-show="totalResults > 0"
+        ref="table"
+        :fields="fields"
+        :items="items"
+        :current-page="currentPage"
+        hover
+        stacked="md"
+        per-page="20"
+        @row-selected="onRowSelected"
+        selectable
+        select-mode="single"
+        :tbody-tr-class="trclass"
+      >
+
+        <template v-slot:cell(Actions)="data">
+          <b-row>
+              <b-col cols="auto" v-if="!data.item.verified.manually">
+                <b-button variant="success" @click="activate(data)" size="sm">Verify</b-button>
+              </b-col>
+              <b-col cols="auto" v-if="!data.item.disabled">
+                <b-button variant="danger" @click="disable(data)" size="sm">Disable</b-button>
+              </b-col>
+              <b-col cols="auto" v-if="data.item.disabled">
+                <b-button variant="warning" @click="enable(data)" size="sm">Enable</b-button>
+              </b-col>
+          </b-row>
+        </template>
+
+        <template v-slot:row-details="row">
+          <b-card>
+            <b-row>
+              <b-col cols="auto" v-if="!row.item.verified.mail">
+                <b-button variant="success" @click="resendMail(row)" size="sm">Resend activation mail</b-button>
+              </b-col>
+              <b-col cols="auto">
+                <a :href="'mailto:' + row.item.contact.email"><b-button variant="primary" size="sm">Send Mail to user</b-button></a>
```

Comment:
These tests should also be internationalised

Reply:
Just UI for admins (us) we all understand english, out of scope for now.

Reply:
Ach, ja.